edge/pkg/devicetwin/dttype/types_helper.go

```diff
@@ -81,6 +80,7 @@ func DeviceTwinToMsgTwin(deviceTwins []dtclient.DeviceTwin) map[string]*MsgTwin
 			actualValue := &TwinValue{Value: &actual}
 			if twin.ActualMeta != "" {
 				json.Unmarshal([]byte(twin.ActualMeta), &actualMeta)
+				actualValue.Metadata = &actualMeta
```

Comment:
Looks good, any bugs will caused? Can we add some tests? :)

Reply:
Seems to be no problem in our environment